xla/python/weakref_lru_cache.cc

```diff
@@ -301,6 +300,15 @@ class WeakrefLRUCache : public std::enable_shared_from_this<WeakrefLRUCache> {
     deferred_deletes.clear();
   }
 
+protected:
```

Comment:
`private:`

We don't expect this class to be subclassed, so I think just say `private`.